tests/operators/test_sql.py

```diff
@@ -322,7 +359,7 @@ def test_pass_min_value_max_sql(self, mock_get_db_hook):
 
         operator.execute()
 
-    @mock.patch.object(ThresholdCheckOperator, "get_db_hook")
+    @mock.patch.object(SQLThresholdCheckOperator, "get_db_hook")
```

Comment:
Was this name changed?

Reply:
Yeah. The original operators were moved to another module and renamed but the tests were forgotten to refactor I believe.

Reply:
You can see the original imports and the refactored imports at the top of the file - `test_sql.py`

Reply:
> tests were forgotten to refactor I believe.

That was what I though, wanted to check. Thanks!